index.js

```diff
@@ -81,7 +81,7 @@ async function underPressure (fastify, opts) {
       ...opts.exposeStatusRoute.routeOpts,
       url: opts.exposeStatusRoute.url,
       method: 'GET',
-      schema: {
+      schema: Object.assign({}, opts.exposeStatusRoute.routeSchemaOpts, {
```

Comment:
Can you add a test for this change?

Reply:
Oh sorry, I only added the type check but forget the normal javascript checking

Reply:
test case added